patches/chromium/build_libc_as_static_library.patch

```diff
@@ -0,0 +1,46 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: VerteDinde <keeleymhammond@gmail.com>
+Date: Wed, 12 May 2021 13:19:19 -0700
+Subject: build_libc++_as_static_library
+
+Build libc++ as static library to compile and pass
+nan tests
```

Comment:
Can we expand this description somewhat? In particular, per https://github.com/electron/electron/blob/master/docs/development/patches.md#patch-justification, I'd like to know:

1. Do we expect to ever be able to remove this patch? If so, under what circumstances? How will a future maintainer know if it's safe to remove this patch?
2. What specific issue does this patch address?

To get yourself in the appropriate mindset, imagine you're an Electron maintainer five years in the future and the people who wrote and reviewed this patch no longer work on the project (and probably don't remember writing this patch at all). What information would you find useful?

Reply:
This is a very good call out, I'll amend this and make the patch 🙇‍♀️